.vscode/settings.json

```diff
@@ -3,5 +3,11 @@
   "ruby lsp.trace.server": "messages",
   "[ruby]": {
     "editor.defaultFormatter": "Shopify.ruby-lsp",
-  }
+  },
+  "cSpell.languageSettings": [
```

Comment:
Based on the default from the README:
https://github.com/streetsidesoftware/vscode-spell-checker#miscellaneous-dictionaries